src/modules/mod.rs

```diff
@@ -150,3 +151,31 @@ pub fn description(module: &str) -> &'static str {
         _ => "<no description>",
     }
 }
+
+fn undistract_me<'a>(context: &'a Context, module: Option<Module<'a>>) -> Option<Module<'a>> {
```

Comment:
It's not clear to me why this has to be a separate module-creating function instead of just a modification to `cmd_duration`, but if we're going this route, this function should be placed in cmd_duration, not in `mod.rs`.

If at all possible, I would prefer that this PR just modify the behaviour of the cmd_duration module so that we don't have to deal with the cognitive overhead of this wrapping when reading code in the future.

Reply:
At the beginning I wasn't sure where to put it. I moved it to `cmd_duration`.